rocAL/rocAL/include/parameters/parameter_crop.h

```diff
@@ -83,6 +83,7 @@ class CropParam
     Parameter<float>* default_y_drift_factor();
     std::vector<uint32_t> x1_arr_val, y1_arr_val, croph_arr_val, cropw_arr_val, x2_arr_val, y2_arr_val;
     bool _random;
+    bool _is_center_crop = false;
```

Comment:
this is also set in constructor. Please remove initialization from here

Reply:
Resolved